test/e2e/traceflow_test.go

```diff
@@ -1084,11 +1103,28 @@ func testTraceflowIntraNode(t *testing.T, data *TestData) {
 
 // testTraceflowInterNode verifies if traceflow can trace inter nodes traffic with some NetworkPolicies set.
 func testTraceflowInterNode(t *testing.T, data *TestData) {
-	node1 := nodeName(0)
-	node2 := nodeName(1)
+	encapMode, err := data.GetEncapMode()
+	if err != nil {
+		t.Fatalf("Failed to retrieve encap mode: %v", err)
+	}
+	if encapMode != config.TrafficEncapModeNoEncap {
+		// https://github.com/antrea-io/antrea/issues/897
+		skipIfProviderIs(t, "kind", "Skipping inter-Node Traceflow test for Kind because of #897")
```

Comment:
OVS datapath has switched to kernel, all issues related to #897 should be resolved now.

Reply:
Thanks, removed.